README.md

```diff
@@ -20,6 +20,19 @@ __What features does it have?__
 - Responsive layout — works well on your smartphone
 - _.. and more!_
 
+__Why the fork?__
+
+We felt that the original [Shout](https://github.com/erming/shout) project
+"stagnated" a little because its original author wanted it to remain his pet
+project (which is a perfectly fine thing!).
+
+A bunch of people, excited about making things a bit differently than the upstream
```

Comment:
Either "doing things" or "a bit different" would sound better, I think, but there's nothing technically wrong with the existing phrasing.